### ISSUE TITLE
Migrate onboarding users beyond booking policies

### ISSUE THREAD

**Issue by situatedbit** [loconomics/loconomics#399]
Part of https://github.com/loconomics/loconomics/issues/402

Since we are removing booking policies from the onboarding process, we need to migrate the data of users who stopped onboarding at booking policies. In the database, move them to the next step.

- [x] Set all existing users to booking request and cancellation policy 1 (?)

**Comment by situatedbit** [comment]
1

**Comment by IagoSRL** [comment]
2

**Comment by joshdanielson** [comment]
1

**Comment by joshdanielson** [comment]
1

**Comment by situatedbit** [comment]
Blocked on #433

**Comment by IagoSRL** [comment]
Added script at is584-epic-onboarding for the migration and executed against *dev* for QA (there was 83 records affected).

Details: it affects both users at bookingPolicies and any user in steps that are not part of the revised Onboarding as of #592, all moved to the new first step publicContactInfo (required since this was before the last step).
